src/Gui/EditableDatumLabel.cpp

```diff
@@ -171,15 +172,15 @@ double EditableDatumLabel::getValue()
     return spinBox->rawValue();
 }
 
-void EditableDatumLabel::setSpinboxValue(double val)
+void EditableDatumLabel::setSpinboxValue(double val, Base::Unit unit)
```

Comment:
Look at the lint just below.  It does not look like we need a copy of the unit, do we? If do not, consider ``` const Base::Unit & unit```. Sorry, I did not catch that one yesterday.